src/settings.ts

```diff
@@ -18,7 +18,7 @@ import sparkleNavLogo from "assets/icons/sparkle-nav-logo.png";
 import defaultMapIcon from "assets/icons/default-map-icon.png";
 import sparkleverseLogo from "assets/images/sparkleverse-logo.png";
 
-export const SPARKLE_HOMEPAGE_URL = "/in/explore/";
+export const SPARKLE_HOMEPAGE_URL = "https://sparklespaces.com/";
```

Comment:
This isn't the correct fix here. We want a seperate new const to be used for the footer so that all of the other places this variable are used don't get changed.

Reply:
`SPARKLE_HOMEPAGE_URL` is not used anywhere in the app except this file

Reply:
@alexey-petrov Yes, but it's used for `HOMEPAGE_URL` which is used in a couple of places. The footer component should be using a separate const, I think that's what Glenn is trying to say.

Reply:
> The footer component should be using a separate const, I think that's what Glenn is trying to say.

Correct. As per the context/discussion on slack where this task was first started from.

Reply:
@denisdimitrov done c69590c